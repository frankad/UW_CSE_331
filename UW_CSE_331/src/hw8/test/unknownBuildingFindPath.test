# Verify that when two unknown building
# is entered by the user to find the path.So, that it will print "Unknown 
# building" followed by the building abbreviated name. For instance, it will
# print once "Unknown building:  AA". it will print twice, "Unknown building:"
# followed by Abbreviated name.
r
ABX
ABY

# terminate the program
q