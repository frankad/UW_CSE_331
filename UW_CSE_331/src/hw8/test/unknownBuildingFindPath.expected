Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify that when two unknown building 
# is entered by the user to find the path.So, that it will print "Unknown 
# building" followed by the building abbreviated name. For instance, it will
# print once "Unknown building:  AA". it will print twice, "Unknown building:"
# followed by Abbreviated name.
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: ABX
Unknown building: ABY

Enter an option ('m' to see the menu): 
# terminate the program